bluesky/plan_stubs.py

```diff
@@ -743,6 +748,12 @@ def inner_trigger_and_read():
             if hasattr(obj, 'trigger'):
                 no_wait = False
                 yield from trigger(obj, group=grp)
+        yield from create(streaming_name)
+        for obj in devices:
+            reading = (yield from read_streaming(obj))
+            if reading is not None:
+                ret.update(reading)
+        yield from save()
```

Comment:
In case anyone else wonders...

I had to remind myself what happens if we 'create' and 'save' with no 'read' messages between. The RunEngine just treats that as a no-op. (So this code block behaves fine if no devices implement `read_streaming`.)

Reply:
yep, thanks for writing this, found it by reading `_save` before writing this :)